### DISCUSSION TITLE
模糊音 - 地球拼音

### DISCUSSION THREAD

**Discussion by lin-ycv** [rime/home#1759] [Q&A]
地球拼音有可能用模糊音嗎？
我嘗試將
https://gist.github.com/lotem/2320943
存為 `%APPDATA%\Rime\terra_pinying.custom.yaml`
但輸入會變的很奇怪：
![image](https://github.com/user-attachments/assets/f382efc0-45ab-42cb-9db5-f001f86e8f0c)
正常沒放`terra_pinying.custom.yaml`時：
![image](https://github.com/user-attachments/assets/77423e2a-f6ae-414f-9623-4e12dd13fde5)

**Comment by lotem** [comment]
朙月拼音和地球拼音的編碼不同. 拼寫運算規則決定輸入碼如何解讀.

**Reply by lin-ycv** [reply]
所有有地球拼音可以用的模糊音規則嗎？
